meta/samples_repository/README.md

```diff
@@ -0,0 +1,102 @@
+# Best practices using the samples repository
+
+There are several ways to access the playbooks included in the samples GitHub
+repository. You can access them using  a web browser, graphical interface (GUI),
+command line interface (CLI() or an ansible playbook, to name a few. We
+recommend that you fork or clone the repository locally to the ansible
+controller, this creates copies of all the playbooks allowing you to try
+them out and customize as necessary.
+
+## Watching the GitHub repository
+The [samples repository](https://github.com/IBM/z_ansible_collections_samples/)
+is a collection of samples that will frequently have new playbooks contributed.
+
+
+In order to know when updates are made, you can **watch** the repository
+and by doing so, you will receive notifications when changes
+are made. In the upper right hand corner you will see the **watch** toggle:
+
+<img width="325" alt="image" src="https://media.github.ibm.com/user/15905/files/97b26800-f2dc-11ea-8e96-f7723979c03e">
+
+## Cloning the repository
+
+Cloning the repository will clone the GitHub repository locally such that you
+will have your own copy and can customize and tailor the samples to your
+specific requirements.
+
+There are a number of ways to clone a repo, command line interface, graphical
+interfaces and even playbooks. Here, will discuss command line interfaces and a
+playbook which when run will either clone or pull the latest changes from the
```

Comment:
Slight grammar suggestion:
`playbook which, when run, will either clone or pull the latest changes ...`